.github/workflows/docker_deploy.yml

```diff
@@ -0,0 +1,31 @@
+name: "Despliegue docker"
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    paths:
+      - "Dockerfile"
```

Comment:
Esto lo acabo de ver, no sé si vendrá del anterior y no lo he visto. ¿Sólo se crea el contenedor si cambia el Dockerfile? ¿Ningún otro fichero que pueda variar y alterar la imagen?

Reply:
No, eso lo tengo del objetivo anterior, lo he puesto por si se modifica algo del docker file crea el contenedor, pero con con cualquier commit de un pull request de la main también, aunque acabo de caer que solo sirve si toco el dockerfile desde la rama main, lo cual no se si es buena idea después de ver la utilidad de las ramas y los pull request

Reply:
Por favor, miraros el manual de las cosas que uséis. Has creado dos filtros diferentes: si se hace pull request, se va a generar siempre (en realidad, nunca, porque desde un PR no tienes acceso a los secrets). Si haces push *en cualquier rama* sólo si cambias ese fichero. Y si cambias una dependencia, ¿qué pasa? ¿No se genera?

Reply:
Me pongo con ello :ok_hand: